samples/NativeModuleSample/csharp/windows/NativeModuleSample/packages.lock.json

```diff
@@ -139,7 +139,10 @@
         "contentHash": "qF6RRZKaflI+LR1YODNyWYjq5YoX8IJ2wx5y8O+AW2xO+1t/Q6Mm+jQ38zJbWnmXbrcOqUYofn7Y3/KC6lTLBQ=="
       },
       "common": {
-        "type": "Project"
+        "type": "Project",
+        "dependencies": {
+          "boost": "[1.76.0, )"
+        }
```

Comment:
Not sure about this change - didn't create any issues when building the sample but wanted to confirm if this should be kept.

Reply:
May just be from a package dependency, if it didn't break the project it's fine to keep :)